compose/docker-compose-local-unittests.yaml

```
@@ -0,0 +1,12 @@
+---
+
+version: '3'
+
+services:
+  database:
+    image: docker.io/postgres:14
+    container_name: unittests
+    environment:
+      POSTGRES_PASSWORD: password
+    ports:
+      - "5001:5432"
```

What if we also had the app here but gave it a profile of "serve" so that `docker compose --profile serve up` would run the db and app and `docker compose up` would only run the db?

```suggestion
---
services:
  database:
    image: docker.io/postgres:14
    container_name: rctab_db
    environment:
      POSTGRES_PASSWORD: password
    volumes:
      - ./postgres_data:/var/lib/postgresql/data
  app:
    build:
      context: ..
      dockerfile: Dockerfile
    container_name: rctab_app
    environment:
      DB_HOST: rctab_db
      DB_PASSWORD: password
      DB_USER: postgres
      MAX_WORKERS: 1
    env_file:
      - ../.env
      - ../.auth.env
    ports:
      - "8000:80"
    profiles:
      - serve
```